exchanges/okx/okx_types.go

```diff
@@ -496,11 +496,6 @@ type DiscountRateInfoItem struct {
 	DiscountCurrencyEquity types.Number `json:"disCcyEq"`
 }
 
```

Comment:
Again, something that can be skipped but given: 
> Fix AccountConfiguration slice return

I get `authenticated request failed invalid value for Number type: VIP1` as a response due to the `AccountConfigurationResponse` `Level` property being `types.Number`

An optional "while you're there, protect the VIP class 🎩"

Reply:
Punted to #1854

Reply:
The level issue has been fixed in #1834 plus some other things to do with account configuration